test/integration/service_account_test.go

```diff
@@ -106,87 +106,6 @@ func TestServiceAccountAuthorization(t *testing.T) {
 	if err := writeClientConfigToKubeConfig(cluster1SAClientConfig, cluster1SAKubeConfigFile.Name()); err != nil {
 		t.Fatalf("error creating kubeconfig: %v", err)
 	}
-
-	// Set up cluster 2 to run against cluster 1 as external kubernetes
```

Comment:
@enj that is correct, 1.6 will drop the external kube entirely and we don't want to take effort fixing api groups for something we dropping ;-)